### PR TITLE
Cplex miqp

### PR DESCRIPTION
**Pull request by tpfau**
Adding a MIQP interface for ibm_cplex. 
Removing the MIQP flag from cplex_direct (its not present in the solveCobraMIQP file).

**I hereby confirm that I have:**

- [X] Tested my code on my own machine
- [X] Followed the guidelines in the [Contributing Guide](https://opencobra.github.io/cobratoolbox/docs/contributing.html)
- [X] Selected `develop` as a target branch (top left drop-down menu)

*(Note: You may replace [ ] with [X] to check the box)*

### CODE REVIEW

File: src/base/solvers/solveCobraMIQP.m [hunk 1]
```diff
@@ -278,6 +280,87 @@
...
+            b_L = MILPproblem.b_L;
+            b_U = MILPproblem.b_U;
+        else
+            b_L = b;
+            b_U = b;
+        end
+        intVars = (vartype == 'B') | (vartype == 'I');
+        % intVars
+        % pause;
+        cplexlp.Model.A = A;
```
Comment (laurentheirendt) [inline src/base/solvers/solveCobraMIQP.m]: Is this code from `solveCobraLP`? Why not reuse it?

Comment (tpfau) [reply src/base/solvers/solveCobraMIQP.m]: Its essentially code from solveCobraMILP.
But yes, makes sense to make a "buildCplexLP" function. I'll have a look at that.

Comment (laurentheirendt) [reply src/base/solvers/solveCobraMIQP.m]: Yes, that would be nice - that will make the code even more maintainable.

**Comment by tpfau** [comment]
closing for various updates.

**Pull request closed without merging**